.buildkite/setup_al2.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+source ./.buildkite/al2env.sh
+
+echo "Creating necessary directories"
+
+mkdir -p $dir
+mkdir -p $bin_path
+mkdir -p $devmapper_path
+mkdir -p $state_path
+
+echo "Resetting thinpool $uuid"
+./tools/thinpool.sh reset $uuid
+
+export INSTALLROOT=$dir
+export FIRECRACKER_CONTAINERD_RUNTIME_DIR=$dir
+echo "Running make"
+sudo make
+echo "Running make install"
+sudo -E "INSTALLROOT=$INSTALLROOT" make install
+echo "Running make install-default-vmlinux"
+sudo make install-default-vmlinux
+echo "Running make image"
+sudo make image
+echo "Running make install-default-rootfs"
+sudo make install-default-rootfs
```

Comment:
Can we remove `sudo` from non-install targets? They should work without being root.

Reply:
Done